synctl/cli.py

```diff
@@ -502,6 +512,93 @@ def pop_size_estimate(self):
         except ValueError as e:
             print(f"Exception: {e}")
 
+    def test_exec_estimate(self, tests, frequency, loc):
+        #The total test executions per month(30 days)
+        exec = tests * ((30 * 24 * 60) / frequency) * loc
```

Comment:
exec is a built-in function, see this https://docs.python.org/3/library/functions.html#exec
```suggestion
       return tests * ((30 * 24 * 60) / frequency) * loc
```